blog/content/post/2020-04-16-3mdeb-contrib-Q1.md

```diff
@@ -0,0 +1,259 @@
+    Michał Żygowski    | amd/pi              | Refactor IVRS generation | [link][25]
+    Michał Żygowski    | amd/pi              | Add lost options | [link][35]
+    Piotr Kleinschmidt | amd/{agesa,pi}      | Change default SATA mode to AHCI | [link][3]
+    Michał Żygowski    | amd/{agesa,pi}      | Include thermal zone | [link][13]
+    Michał Żygowski    | amd/{agesa,pi}      | Use ACPIMMIO common block wherever possible | [link][33]
+    Michał Żygowski    | amdblocks/acpimmio  | Add missing MMIO functions | [link][32]
+    Michał Żygowski    | mb/*                | Use ACPIMMIO common block wherever possible | [link][37]
+    Michał Żygowski    | acpi                | Correct the processor devices scope | [link][20]
+    Michał Żygowski    | x86/acpi            | Add definitions for IVHD type 11h | [link][24]
+    Michał Żygowski    | drivers/pc80/tpm    | Change the _HID and _CID for TPM2 device | [link][26]
+    Michał Żygowski    | maintainers         | Add 3mdeb as Protectli mainboards maintainers | [link][8]
+    Michał Żygowski    | protectli/vault     | Add FW2B and FW4B Braswell based boards support | [link][7]
+    Michał Żygowski    | protectli/vault_kbl | Add FW6 support | [link][10]
+    Michał Żygowski    | libretrend/lt1000   | Add Libretrend LT1000 mainboard | [link][9]
+    Michał Żygowski    | superio/nuvoton     | Add chip config option to reset GPIOs | [link][22]
+    Piotr Kleinschmidt | superio/nuvoton     | Add virtual LDN for simple GPIO IO control | [link][30]
+    Piotr Kleinschmidt | superio/nuvoton     | Add soft reset GPIO functionality | [link][38]
+    Michał Żygowski    | intel/bd82x6x       | Configure CLKRUN_EN according to SKU | [link][28]
+    Michał Żygowski    | intel/braswell      | Generate microcode binaries from tree | [link][5]
+    Michał Żygowski    | intel/braswell      | Include smbios.h for Type9 Entries | [link][6]
+
+    ---
+
+    ![TrenchBoot logo](/covers/trenchboot-logo.png)
+
+2. [TrenchBoot/landing-zone](https://github.com/TrenchBoot/landing-zone/)
+
+    > TrenchBoot is a framework that allows individuals and projects to build
+    > security engines to perform launch integrity actions for their systems.
+    > The framework builds upon Boot Integrity Technologies (BITs) that
+    > establish one or more Roots of Trust (RoT) from which a degree of
+    > confidence that integrity actions were not subverted.
+
+    Full list of 2020'Q1 patches:
+
+    Author          | Category | Patch | URL
+    ----------------|----------|-------|----
+    Krystian Hebel  | Build    | Move bootloader data out of measured block | [link][39]
+    Krystian Hebel  | Build    | Use more hidden symbols to fix 32bit boot | [link][40]
+    Michał Żygowski | Security | Add sha256 | [link][41]
+    Michał Żygowski | README   | Add basic readme with Travis build status | [link][42]
+    Krystian Hebel  | Main     | Move PCR extension logic to a separate function | [link][43]
```

Comment:
@artur-rs I though there was my patch also, but may it was merged before Q1?

Reply:
@pietrushnic it is marked as `7/22/2019`